ota-plus-web/app/reactapp/src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HERE OTA Connect</title>
+    <meta name="description" content="HERE OTA Connect">
+    <meta name="keywords" content="garage">
+    <link rel="icon" href="/assets/img/icons/favicon.ico" type="image/x-icon" />
+    <!-- <link href="../style/style.scss" rel="stylesheet" type="text/css"/> -->
+  </head>
+  <body>
+    <form id="logout" action="/logout">
+    </form>
+    <input type="hidden" id="csrf-token-val" value="@csrf.get.value"/>
+    <input type="hidden" id="ws-url" value="@wsUrl"/>
+    <input type="hidden" id="toggle-atsGarageTheme" value="@{uiToggles.atsGarageTheme}"/>
+    <input type="hidden" id="toggle-autoFeatureActivation" value="@{uiToggles.autoFeatureActivation}"/>
+    <!-- Hides user dropdown -->
```

Comment:
This looks similar to what was removed in 706854528bd61d379fe31d2a6926573d0d196b6f